homeassistant/components/xiaomi_miio/vacuum.py

```diff
@@ -67,6 +70,15 @@
     vol.Optional(ATTR_RC_DURATION): cv.positive_int,
 })
 
+SERVICE_SCHEMA_CLEAN_ZONE = VACUUM_SERVICE_SCHEMA.extend({
+    vol.Required(ATTR_ZONE_ARRAY):
+        vol.All(list, [vol.ExactSequence(
+            [vol.Coerce(int), vol.Coerce(int),
+             vol.Coerce(int), vol.Coerce(int)])]),
+    vol.Required(ATTR_ZONE_REPEATER):
+        vol.All(vol.Coerce(int), vol.Clamp(min=1, max=3)),
```

Comment:
Is this clamping due to hardware limitations? If yes, it's worth adding a note about it somewhere in the docs?

Reply:
Zone must be defined as array of 4 integer values. 
In documentation this is explained.
![image](https://user-images.githubusercontent.com/5739630/53093624-38cda800-3518-11e9-96e9-7f1e61e4cba4.png)

Reply:
The question is if I can request four rounds in one go, or is this limited by the device?

Reply:
you must pass the zones defined as an array of integer values to the module.
https://python-miio.readthedocs.io/en/latest/_modules/miio/vacuum.html?highlight=zoned_clean

you can define more zone to clean in one go

Reply:
I'm talking about `repeats` option... But I think this PR can be merged now, thanks for your contribution! :1st_place_medal: